app/views/declaration/declarationitems/items_add_item.scala.html

```diff
@@ -40,17 +41,36 @@
 
 
 @govukLayout(
-  title = Title("declaration.itemsAdd.title", "declaration.section.5"),
+    title =
+        if (request.isType(CLEARANCE)) Title("declaration.clearance.itemsAdd.title", "declaration.section.5")
+        else Title("declaration.itemsAdd.title", "declaration.section.5"),
   backButton = Some(BackButton(messages("site.backToPreviousQuestion"), navigator.backLink(ExportItem)))) {
 
   @formHelper(action = routes.ItemsSummaryController.addFirstItem, 'autoComplete -> "off") {
 
     @sectionHeader(messages("declaration.section.5"))
-    @pageTitle(messages("declaration.itemsAdd.title"))
+    @pageTitle(if (request.isType(CLEARANCE))
+                    messages("declaration.clearance.itemsAdd.title")
+                    else messages("declaration.itemsAdd.title")
+    )
 
-    @paragraphBody(messages("declaration.itemsAdd.paragraph.1"))
-    @paragraphBody(messages("declaration.itemsAdd.paragraph.2"))
-    @paragraphBody(messages("declaration.itemsAdd.paragraph.3"))
+    @paragraphBody(
+        if (request.isType(CLEARANCE))
+            messages("declaration.clearance.itemsAdd.paragraph.1")
+        else
+            messages("declaration.itemsAdd.paragraph.1")
+    )
+    @paragraphBody(
+        if (request.isType(CLEARANCE))
+            messages("declaration.clearance.itemsAdd.paragraph.2")
+        else
+            messages("declaration.itemsAdd.paragraph.2")
+    )
+    @paragraphBody(
+        if (!request.isType(CLEARANCE))
+            messages("declaration.itemsAdd.paragraph.3")
+        else ""
+    )
```

Comment:
IMO, this implementation is a bit messy. It could be simplified with something like...
```
@if(request.isType(CLEARANCE)) {
    @govukLayout(
        ...
    )
} else {
    @govukLayout(
        ...
    )
}
```

Reply:
Resolved as advised.